.gitignore

```diff
@@ -3,3 +3,11 @@ backend/workingdir/
 client/build
 client/node_modules/
 documentation/_build
+/docker-compose.yml
+/Dockerfile
+/**/client/package.json
+/**/client/proxy.conf.json
+/client/.angular
+.scannerwork/
+.vscode/
```

Comment:
This file has been edited across multiple commits with instances of code being removed and then re-added. To streamline the review, the offending commits should be removed, either by rebasing or squashing them into a single, meaningful commit.